fe/fe-core/src/main/java/com/starrocks/common/proc/FrontendsProcNode.java

```diff
@@ -46,7 +46,7 @@ public class FrontendsProcNode implements ProcNodeInterface {
 
     public static final ImmutableList<String> TITLE_NAMES = new ImmutableList.Builder<String>()
             .add("Name").add("IP").add("EditLogPort").add("HttpPort").add("QueryPort").add("RpcPort")
-            .add("Role").add("IsMaster").add("ClusterId").add("Join").add("Alive").add("ReplayedJournalId")
+            .add("Role").add("ClusterId").add("Join").add("Alive").add("ReplayedJournalId")
```

Comment:
I think you should keep isMaster in this PR, then this PR don't change user interface. we can merge this PR quickly.
And you make this change in another PR.

Reply:
done